### PR TITLE
Added Gloo as an example of an ingress controller

### PR DESCRIPTION
**Pull request by christian-posta**
Gloo is a function-based proxy based on Envoy which adds ingress and API Gateway functionality. Would like to add this to the list of ingress controllers in the ingress docs.

### CODE REVIEW

File: content/en/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Additional controllers include:
 
 * [Contour](https://github.com/heptio/contour) is an [Envoy](https://www.envoyproxy.io) based ingress controller
   provided and supported by Heptio.
+* [Gloo](https://gloo.solo.io) is an open-source ingress controller based on [Envoy](https://www.envoyproxy.io) which offers API Gateway functionality with enterprise support from [solo.io](https://www.solo.io)
 * F5 Networks provides [support and maintenance](https://support.f5.com/csp/article/K86859508)
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/ingress.md]: @christian-posta, are the entries listed alphabetically? To be consistent, you could add a period at the end.

Comment (christian-posta) [reply content/en/docs/concepts/services-networking/ingress.md]: Good catch @kbhawkey I've pushed an update

**Pull request merged**